src/flow/utils.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import traverse from "babel-traverse";
+import { BabelNode } from "babel-types";
+import * as t from "babel-types";
+
+// taken directly from Babel:
+// https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-flow-strip-types/src/index.js
+export function stripFlowTypeAnnotations(ast: BabelNode): void {
```

Comment:
This is all taken from Babel

Reply:
Can we just use the code from the upstream package rather than copy then?

Reply:
If we can't use the code as is by reference, we should document the exact differences but we also need to include copyright header (which is McKenzie I guess).

Reply:
We can't use it from upstream but I'll make make those changes in another PR.